### PR TITLE
Remove deprecated locks

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Part of #1391.

### CODE REVIEW

File: runtime/crab/src/migration.rs [hunk 1]
```diff
@@ -41,47 +41,33 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
...
+		if let Ok(acct) = array_bytes::hex_n_into::<_, AccountId, 20>(acct) {
+			<Locks<Runtime>>::mutate(acct, |ls| {
+				ls.retain(|l| &l.id != *lid);
+			});
+		}
 	});
 
 	// frame_support::weights::Weight::zero()
-	<Runtime as frame_system::Config>::DbWeight::get().reads_writes(0, w)
+	<Runtime as frame_system::Config>::DbWeight::get().reads_writes(0, 26)
```
Comment (boundless-forest) [inline runtime/crab/src/migration.rs:72]: Can you add some note/comments about the concrete read or write operation here? Seems two writes for single item.

File: runtime/darwinia/src/migration.rs [hunk 2]
```diff
@@ -42,20 +42,27 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
...
+		if let Ok(acct) = array_bytes::hex_n_into::<_, AccountId, 20>(acct) {
+			<Locks<Runtime>>::mutate(acct, |ls| {
+				ls.retain(|l| &l.id != *lid);
+			});
+		}
+	});
 
 	// frame_support::weights::Weight::zero()
-	<Runtime as frame_system::Config>::DbWeight::get().reads_writes(0, 5)
+	<Runtime as frame_system::Config>::DbWeight::get().reads_writes(0, 16)
```
Comment (boundless-forest) [inline runtime/darwinia/src/migration.rs:67]: The same as the above comment.

File: runtime/crab/src/migration.rs (hunk 1 above)
Comment (AurevoirXavier) [reply runtime/crab/src/migration.rs:72]: `Balance::Locks + System::Account`

Comment (AurevoirXavier) [reply runtime/crab/src/migration.rs:72]: Simply `Balances::Lock::writes() * 2`. 

It's kind of arbitrated choice.
Leave some redundant space.

**Review by boundless-forest — APPROVED** [review]

**Pull request merged**